tests/rip7560/test_send_failed.py

```diff
@@ -36,3 +39,82 @@ def test_eth_send_no_code_wrong_nonce(w3):
 
     ret = tx.send()
     assert_rpc_error(ret, "nonce too high", RPCErrorCode.INVALID_INPUT)
+
+
+def test_eth_send_account_validation_reverts(wallet_contract_rules, tx_7560):
+    tx_7560.sender = wallet_contract_rules.address
+    tx_7560.nonce = hex(2)
+    tx_7560.signature = to_prefixed_hex("revert-msg")
+
+    response = tx_7560.send()
```

Comment:
testing the wrong function: 
normal transaction don't return revert string in "eth_send", but only in "eth_call" (and "eth_estimate")

These are the methods that need to be checked (but first, we need to implement them)

Reply:
Well, I think there is a difference between legacy and 7560 transactions in that we do want to report the reason for transaction being invalid. This **does** happen with legacy transactions, if you send one with invalid nonce or signature you will get an exception right in the `eth_send`. 
On the other hand, `eth_call` should provide the "returned result" of the execution frame, if there was any. But also, why would there be such a thing? I don't see `eth_call` being useful in the context of RIP-7560.